packages/create-toolpad-app/index.ts

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+import { execaCommand } from 'execa';
+import inquirer from 'inquirer';
+import fs from 'fs';
+import { getPackageManager } from './utils/getPackageManager';
+
+// Detect the package manager
+const packageManager = getPackageManager();
+
+// Create a new directory and initialize a new project
+const scaffoldProject = async () => {
+  const name = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'projectName',
+      message: 'Enter the name of your project:',
+      default: 'my-app',
+    },
+  ]);
+
+  // eslint-disable-next-line no-console
+  console.log(`Creating a new MUI Toolpad project in ${chalk.blue(name.projectName)}`);
+  try {
+    fs.mkdirSync(name.projectName);
+    process.chdir(name.projectName);
+
+    const packageJson = {
```

Comment:
Could we generate this by copying a package.json file already configured by us instead? I feel like that could be better to work with - it's what I had tried in https://github.com/mui/mui-toolpad/pull/1705. 
On the other hand this approach could be more customizable with Javascript, so anyway just bringing up the possibility - it's probably still too early to tell what option would be best at this point.